app/views/cluster_stats/index.html.erb

```diff
@@ -1,16 +1,313 @@
-<%= render 'validators/banner' %>
-<% unless @mainnet.empty? %>
-  Mainnet Stats: <br />
-  <%= @mainnet.inspect %>
-  <%#= render 'validators/stats' %>
+<h1 class="page-header"><%= params[:network].capitalize %> Stats</h1>
 
-<% end %>
 
-<% unless @testnet.empty? %>
-  Testnet Stats: <br />
-  <%= @testnet.inspect %>
-  <%#= render 'validators/stats' %>
+<div class="card">
+  <div class="card-content">
+    <h3 class="card-heading mb-3">Top Validators by:</h3>
+  </div>
 
-<% end %>
+  <ul class="nav nav-tabs">
+    <li class="nav-item">
+      <a class="nav-link active" data-toggle="tab" href="#menu1">Stake</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#menu2">Skipped Vote&nbsp;&percnt;</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#menu3">Root Distance</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#menu4">Vote Distance</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#menu5">Skipped Slot&nbsp;&percnt;</a>
+    </li>
+  </ul>
 
-<%= render 'validators/software_versions' %>
+  <!-- Tab panes -->
+  <div class="tab-content">
+    <div class="tab-pane active" id="menu1">
+      <div class="table-responsive-md">
+        <table class='table mb-0'>
+          <thead>
+          <tr>
+            <th>#</th>
+            <th>Validator</th>
+            <th>Stake</th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody>
+          <% @stats[:top_staked_validators].first(50).each.with_index do |v, index| %>
```

Comment:
We want to show only 50 validators? No pagination?

Reply:
TBH I don't know yet how many records we'll want to display, maybe even fewer. So I left it like this for now.